managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedCursor.java

```diff
@@ -86,14 +86,26 @@ enum IndividualDeletedEntries {
      */
     Map<String, String> getCursorProperties();
 
-     /**
-      * Updates the properties.
-      * @param cursorProperties
-      * @return a handle to the result of the operation
-      */
-     default CompletableFuture<Void> setCursorProperties(Map<String, String> cursorProperties) {
-         return CompletableFuture.completedFuture(null);
-     }
+    /**
+     * Add a property associated with the cursor.
+     *
+     * @return a handle to the result of the operation
+     */
+    CompletableFuture<Void> putCursorProperty(String key, String value);
```

Comment:
We should add some notes here (like what exception will be set in this CompletableFuture, if there are concurrent modification) if we want decide to let caller to handle conflict.

Reply:
Ok, I add some notes.